index.js

```diff
@@ -185,6 +187,26 @@ export const consumePurchase = (token) => Platform.select({
   android: async() => RNIapModule.consumeProduct(token),
 })();
 
+/**
+ * Should Add Store Payment (iOS only)
+ *   Indicates the the App Store purchase should continue from the app instead of the App Store. 
+ * @returns {null}
+ */
+export const getPromotedProduct = () => Platform.select({
+  ios: async() => RNIapIos.promotedProduct(),
+  android: async() => Promise.resolve()
+})()
```

Comment:
Hope you but `;` <== at the end to follow other lint.

Reply:
Also, we are using `trailing commas`.

Reply:
It might be worth setting up ESLint and adding your style rules to the repo rather than going through this process each time ;)

Reply:
There is actually defined [here](https://github.com/dooboolab/react-native-iap/blob/master/.eslintrc) however this is pretty outdated and should be updated. However, [comma-dangle](https://github.com/dooboolab/react-native-iap/blob/3914f6f3a3f3a7d69d74f78b2709de70752793ea/.eslintrc#L8) defined here.